VA-Explorer/va_explorer
# Issue174 - separation of warning types on data_management tab

This PR closes [issue 174](https://github.com/VA-Explorer/va_explorer/issues/174)

As it stands, warnings generated by the questionnaire results include warnings from user error and algorithm error. This can be improved by separating them into two different types of warnings.

This implementation uses a regex pattern to detect algorithm warnings, all other warnings are assumed to be user warnings.

## Regex
### Actual
"^W\d{6}[-]"
### Read as
String that starts with "W", 6 digit sequence, and a dash. 
This regex was made based on my observation of what pattern all these warnings share. This regex pattern can be editted later on should the warnings change or any other reason causing algorithm warnings to not be detected.

## Testing procedure:
Checkout into branch issue174, make sure to have populated data with questionnaire warnings/errors.
Spin up the site and navigate to the 'Data Management' tab.
Select to view any row of questionnaire results
Switch to 'Coding Issues' tab and observe changes to be accurate.

Before:
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/33464879/148454382-0e4431a6-ac80-4c24-ac24-9099419d85be.png">

After:
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/33464879/148454314-6f0fd19f-fa06-4da9-9bf6-0b5f2f568ad9.png">
